### PR TITLE
Adapt changelog for versionbot

### PR DESCRIPTION
**Pull request by lekkas**
Change-Type: patch
Fixes: #173

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
----------
+# Change Log
 
-All notable changes to this project will be documented in this file.
+All notable changes to this project will be documented in this file
+automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-### v5.0.22
+## v5.0.22
```
Comment (jviotti) [inline CHANGELOG.md]: Should you do this change for the other entries as well?

Comment (lekkas) [inline CHANGELOG.md]: I'll happily do for consistency reasons, let check with @hedss as well if this is a Versionbot requirement as well

Comment (hedss) [inline CHANGELOG.md]: It's down to `versionist` and I don't think it cares as long as it finds the latest and the tag.

Comment (lekkas) [inline CHANGELOG.md]: Fixes other version headers as well

**Pull request merged**